src/components/ShopItems.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom';
+
+function ShopItems({ data }) {
+
+  const items = data.map( (item, index) => {
+    if (index < 8) {
+      return (
+        <div className="imgItem">
+          <img src={item.image_url} />
+          <p>{item.name}</p>
+          <p className="price">{item.price}</p>
+          <p className={item.shipping.split(' ').join('')}>{item.shipping}</p>
+        </div>
+      );
+    }
+  })
+
+  return (
+    <div className="imgContainer">
+      {items}
+    </div>
+  );
+}

```

Comment:
This is a nice way of only displaying 8 items. Were you still running into the problem where it was mixed up/random on refresh?

Reply:
No I fixed that problem. The shop items will not be random because I'm selecting just the first 8 items that shop has in the database